hack/common.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
-# Location of the manifests file
-MANIFEST_LOC=bundle/
+# Set location of the manifests file
+if [ -z ${COMMUNITY:-} ]; then
+  MANIFEST_LOC=bundle/
+else
+  MANIFEST_LOC="$ARTIFACT_DIR"
+fi
```

Comment:
I would keep a default value and update as necessary. That is:
```shell
# set default location of the manifests file
MANIFEST_LOC="$ARTIFACT_DIR"
# update for commutiy 
if [ -z ${COMMUNITY:-} ]; then
  MANIFEST_LOC=bundle/ 
fi
```

Reply:
@jrvaldes Thanks, but I might keep what I have. ARTIFACT_DIR is not the default location. That if statement checks if COMMUNITY is unset and keeps the default bundle/ location without triggering `set -o nounset`